reana_workflow_controller/rest/utils.py

```diff
@@ -193,10 +193,11 @@ def remove_workflow_jobs_from_cache(workflow):
     :param workflow: The workflow object that spawned the jobs.
     :return: None.
     """
+    workspace_rel_path = os.path.relpath(workflow.workspace_path, SHARED_VOLUME_PATH)
```

Comment:
Why this line is needed? Maybe I'm missing something, but I think it will try to `remove_workflow_workspace()` without `/var/reana` which should be incorrect for some workspaces

Reply:
You are right, as I modified the path in remove_workflow_workspace()